### ISSUE TITLE
outside checkbox to check all the nodes to of checkbox tree.

### ISSUE THREAD

**Issue by aqib1992** [jakezatecky/react-checkbox-tree#143]
I want to check all the checkbox with outside checkbox.
It is possible now ??
If not will you consider doing that ??

**Comment by jakezatecky** [comment]
There currently does not exist an option for a checkbox outside of the main tree to toggle all nodes. The best way to allow a user to check/uncheck all nodes would be to have a single root-level node and add its value to the `expanded` array. That would essentially accomplish the purpose of checking all nodes but it would be within the tree.

I have considered adding a check all/uncheck all option like there is expand all/collapse, but decided against it due to the above solution.

**Comment by aqib1992** [comment]
Yeah i do understand that but my structure is different. I have Multiple tabs each tab contains react checkbox tree. and i refer that tab to whole tree.

**Comment by aqib1992** [comment]
![Screenshot from 2019-04-23 10-34-01](https://user-images.githubusercontent.com/47976440/56556041-73a19a00-65b4-11e9-9c21-78d94348e9e7.jpg)

Plz refer the attach file to see my structure.

**Comment by jakezatecky** [comment]
I would not be opposed to adding `checkAllNodes` and `uncheckAllNodes` methods to the tree instance that you could invoke using a [React ref](https://reactjs.org/docs/refs-and-the-dom.html). However, you would have to create whatever element in the interface to expose this action to the user.

**Comment by aqib1992** [comment]
i am trying to add new function to the library. But below Error Occurs: 
Error: Cannot find module './test/setup.js'.
